docker/runtime/run-container.ps1

```diff
@@ -14,5 +14,5 @@ docker run `
     -p 15581:15581 `
     --network bridge `
     --name iotedgec `
-    -e IOT_DEVICE_CONNSTR=$IOT_DEVICE_CONNSTR `
-    iot-edge-c
+    -e IOT_DEVICE_CONNSTR=$env:IOT_DEVICE_CONNSTR `
```

Comment:
Please help make sure that not double-quoting the connection string won't cause issues.

Reply:
I tried several times and haven't got any issue so far